R/compact_character.R

```diff
@@ -0,0 +1,18 @@
+#' Remove empty strings from character
+#'
+#' @param x A single character or a vector of characters.
+#'
+#' @return
+#'
+#' A character or a character vector with empty strings removed.
+#'
+#' @examples
+#' compact_character(c("x", "y", NA))
+#' compact_character(c("x", NULL, "", "y"))
```

Comment:
no need for NULL here

Reply:
Good point. I was just copying and pasting the list examples and replacing with vectors. Modified.